File: backend/src/main/java/com/backend/domain/user/repository/VerificationCodeRepository.java
package com.backend.domain.user.repository;

import com.backend.domain.user.entity.VerificationCode;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;

public interface VerificationCodeRepository extends JpaRepository<VerificationCode, Long> {
    Optional<VerificationCode> findByEmail(String email);
}

File: backend/src/main/java/com/backend/domain/user/repository/VerificationCodeRepository.kt
package com.backend.domain.user.repository

import com.backend.domain.user.entity.VerificationCode
import org.springframework.data.jpa.repository.JpaRepository

interface VerificationCodeRepository : JpaRepository<VerificationCode, Long> {
    fun findByEmail(email: String): VerificationCode?
}
